com.unity.render-pipelines.core/Runtime/Inputs/InputRegistering.cs

```diff
@@ -66,8 +57,76 @@ static void WriteEntry(SerializedProperty spAxes, InputManagerEntry entry)
             spAxis.FindPropertyRelative("joyNum").intValue = (int)entry.joystick;
         }
 
-        public static void RegisterInputs(List<InputManagerEntry> entries)
+        static void AddEntriesWithoutCheck(SerializedProperty spAxes, List<InputManagerEntry> newEntries)
         {
+            int startRange = spAxes.arraySize;
```

Comment:
note for next time: i got confused because expected a range here. so could you confirm startRange is basically firstEntryIndex and endRange is lastEntryIndex?

Reply:
Yes I was refering it at range like in the list.AddRange(...) method.
So startRange = firstEntryPositionInInputListIndex and endRange = lastEntryPositionInInputListIndex

Just first/lastEntryIndex is unclear if it is in the newEntries list or in the spAxes array. So I'll keep range and add a comment.